app/controllers/users_controller.rb

```diff
@@ -2,12 +2,16 @@
 class UsersController < ApplicationController
   load_permissions_and_authorize_resource
   before_action :set_user
+
   def index
   end
 
   def show
   end
 
+  def profile
```

Comment:
Oh, and `/profile` with no id is not a good idea. One example is that links don't work (checkout *my* profile at fsektionen.se/profile).